test/integration/uniqueness_test.exs

```diff
@@ -36,71 +36,93 @@ defmodule Oban.Integration.UniquenessTest do
 
     assert count_jobs() == 3
   end
 
-  test "scoping uniqueness by period" do
+  test "scoping uniqueness by period", context do
     now = DateTime.utc_now()
     two_minutes_ago = DateTime.add(now, -120, :second)
     five_minutes_ago = DateTime.add(now, -300, :second)
     one_thousand_years_ago = Map.put(now, :year, now.year - 1000)
     ten_years_in_seconds = 100 * 365 * 24 * 60 * 60
 
-    assert %Job{id: _id} = unique_insert!(%{id: 1}, inserted_at: two_minutes_ago)
-    assert %Job{id: _id} = unique_insert!(%{id: 2}, inserted_at: five_minutes_ago)
-    assert %Job{id: _id} = unique_insert!(%{id: 3}, inserted_at: one_thousand_years_ago)
-    assert %Job{id: id_1} = unique_insert!(%{id: 1}, unique: [period: 110])
-    assert %Job{id: id_2} = unique_insert!(%{id: 2}, unique: [period: 290])
-    assert %Job{id: id_3} = unique_insert!(%{id: 3}, unique: [period: ten_years_in_seconds])
+    assert %Job{id: _id} = unique_insert!(context.name, %{id: 1}, inserted_at: two_minutes_ago)
+    assert %Job{id: _id} = unique_insert!(context.name, %{id: 2}, inserted_at: five_minutes_ago)
+
+    assert %Job{id: _id} =
+             unique_insert!(context.name, %{id: 3}, inserted_at: one_thousand_years_ago)
+
+    assert %Job{id: id_1} = unique_insert!(context.name, %{id: 1}, unique: [period: 110])
+    assert %Job{id: id_2} = unique_insert!(context.name, %{id: 2}, unique: [period: 290])
+
+    assert %Job{id: id_3} =
+             unique_insert!(context.name, %{id: 3}, unique: [period: ten_years_in_seconds])
 
-    assert %Job{id: ^id_1} = unique_insert!(%{id: 1}, unique: [period: 180])
-    assert %Job{id: ^id_2} = unique_insert!(%{id: 2}, unique: [period: 400])
-    assert %Job{id: ^id_3} = unique_insert!(%{id: 3}, unique: [period: :infinity])
+    assert %Job{id: ^id_1} = unique_insert!(context.name, %{id: 1}, unique: [period: 180])
+    assert %Job{id: ^id_2} = unique_insert!(context.name, %{id: 2}, unique: [period: 400])
+    assert %Job{id: ^id_3} = unique_insert!(context.name, %{id: 3}, unique: [period: :infinity])
 
     assert count_jobs() == 6
   end
 
-  test "inserting unique jobs within a multi transaction" do
-    assert {:ok, %{job_1: job_1, job_2: job_2, job_3: job_3}} =
-             Multi.new()
-             |> Oban.insert(:job_1, UniqueWorker.new(%{id: 1}))
-             |> Oban.insert(:job_2, UniqueWorker.new(%{id: 2}))
-             |> Oban.insert(:job_3, UniqueWorker.new(%{id: 1}))
-             |> Repo.transaction()
+  test "inserting unique jobs within a multi transaction", context do
+    multi = Multi.new()
+    multi = Oban.insert(context.name, multi, :job_1, UniqueWorker.new(%{id: 1}))
+    multi = Oban.insert(context.name, multi, :job_2, UniqueWorker.new(%{id: 2}))
+    multi = Oban.insert(context.name, multi, :job_3, UniqueWorker.new(%{id: 1}))
+    assert {:ok, %{job_1: job_1, job_2: job_2, job_3: job_3}} = Repo.transaction(multi)
```

Comment:
What a shame, this was a nice pipe chain 😢

Reply:
What can I say. You win some, you lose some :-)